site/docs/v1/tech/client-libraries/netcore.adoc

```diff
@@ -37,11 +37,100 @@ include::docs/v1/tech/client-libraries/_static-patch-note.adoc[]
+
+Currently, the following .NET templates are implemented:
+
+* FusionAuth Blazor Server Application.
+* FusionAuth MVC Application.
+* FusionAuth Web API Application.
+
+
+=== Requirements
+
+* A FusionAuth installation. You can link:/docs/v1/tech/installation-guide/[install FusionAuth locally, or sign up for a hosted account].
+* .NET Core 7.0 SDK or higher.
+* Visual Studio 2022 for Mac, version 17.6 and above (optional).
+* Visual Studio 2022 for Windows, version 17.6 and above (optional).
+
+
+=== Installing the Templates
+
+FusionAuth .NET templates are available on link:https://www.nuget.org/packages/FusionAuth.Templates/[NuGet]. You can install them by running the following command in your terminal.
+
+```bash
+dotnet new install FusionAuth.Templates::1.0.0
+```
+
+When installed successfully, the templates will be available in the .NET CLI and Visual Studio. The installation is the same for both Windows and macOS.
+
+=== Using the Templates
+
+To help you set up a valid application in FusionAuth, we have created a .NET FusionAuth link:/docs/v1/tech/installation-guide/kickstart[Kickstart] script, available on https://github.com/FusionAuth/fusionauth-example-client-libraries/tree/main/dotnet/Kickstart[GitHub]. Follow the instructions in the Github repo to set up FusionAuth for your project. If you don't want to use the Kickstart script, you can set up your application manually in FusionAuth, using the values referenced in the Kickstart script. 
+
+Note that the Kickstart script is designed to run when starting a newly installed FusionAuth instance the first time. The Kickstart file cannot be used on an existing FusionAuth instance. 
+
+The Kickstart script assumes that your .NET project will be running on `HTTPS` on port `5001`. Your project might not run on the same port, as Visual Studio will randomly choose a port if the chosen one is already in use by another project. It may also be a different port if you run the project through IIS or another web server. In this case, update the `authorizedRedirectURL` and `logoutURL` variables in the `kickstart.json` file in `kickstart` directory to that of your project. If you need to, you can also update these in the Application settings page in the FusionAuth admin site in [the OAuth tab](https://fusionauth.io/docs/v1/tech/core-concepts/applications#oauth) at any time after the Kickstart has run.
+
+Some templates will ask for a FusionAuth Client Secret when initializing a new project. Use a non-sensitive secret from a local FusionAuth installation. If you use the setup script to set up your FusionAuth instance, the Client Secret will be `change-this-in-production-to-be-a-real-secret`.
```

Comment:
What script are you talking about?  Is this the something other than the kickstart?